tests/__init__.py

```diff
@@ -1507,3 +1507,156 @@ def pay(self):
+                            "address_postal_code_check": "pass",
+                            "cvc_check": "pass",
+                        },
+                        "country": "US",
+                        "exp_month": 10,
+                        "exp_year": 2020,
+                        "fingerprint": "sb2OAOijRKy8wYHu",
+                        "funding": "credit",
+                        "last4": "4242",
+                        "three_d_secure": None,
+                        "wallet": None,
+                    },
+                    "type": "card",
+                },
+                "receipt_email": "kyoung@hotmail.com",
+                "receipt_number": None,
+                "receipt_url": (
+                    "https://pay.stripe.com/receipts/acct_1DrGYIB7kbjcJ8Qq/"
+                    "ch_1FG74WB7kbjcJ8Qqx1oIdqfG/rcpt_FleN33oToRTXKCy6sxd5Stnh0ttnxYT"
+                ),
+                "refunded": False,
+                "refunds": {
+                    "object": "list",
+                    "data": [],
+                    "has_more": False,
+                    "total_count": 0,
+                    "url": "/v1/charges/ch_1FG74WB7kbjcJ8Qqx1oIdqfG/refunds",
+                },
+                "review": None,
+                "shipping": None,
+                "source": None,
+                "source_transfer": None,
+                "statement_descriptor": "FOO DESCRIPTOR",
+                "statement_descriptor_suffix": None,
+                "status": "succeeded",
+                "transfer": "tr_16Y9BK2eZvKYlo2CR0ySu1BA",
+                "transfer_data": {
+                    "amount": None,
+                    "destination": "acct_1032D82eZvKYlo2C",
+                },
+                "transfer_group": "group_pi_1FG742B7kbjcJ8QqGKF6qIM0",
+            }
+        ],
+        "has_more": False,
+        "total_count": 1,
+        "url": "/v1/charges?payment_intent=pi_1FG742B7kbjcJ8QqGKF6qIM0",
+    },
+    "client_secret": "pi_1FG742B7kbjcJ8QqGKF6qIM0_secret_yeRoAechksXUy2HdUydIKlGbw",
+    "confirmation_method": "automatic",
+    "created": 1567874826,
+    "currency": "usd",
+    "customer": "cus_6lsBvm5rJ0zyHc",
+    "description": "Online payment for FOO",
+    "invoice": None,
+    "last_payment_error": None,
+    "livemode": False,
+    "metadata": {"foo": "bar"},
+    "next_action": None,
+    "on_behalf_of": "acct_1032D82eZvKYlo2C",
+    "payment_method": "pm_fakefakefakefake0001",
+    "payment_method_options": {"card": {"request_three_d_secure": "automatic"}},
+    "payment_method_types": ["card"],
+    "receipt_email": "kyoung@hotmail.com",
+    "review": None,
+    "setup_future_usage": None,
+    "shipping": None,
+    "source": None,
+    "statement_descriptor": "FOO DESCRIPTOR",
+    "statement_descriptor_suffix": None,
+    "status": "succeeded",
+    "transfer_data": {"destination": "acct_1032D82eZvKYlo2C"},
+    "transfer_group": "group_pi_1FG742B7kbjcJ8QqGKF6qIM0",
+}
```

Comment:
Would be nice to move this to a generated fixture, I'll have a look at how feasible it'd be to update the generate fixtures command to handle it.

Reply:
Actually it's probably not sensible to try to have `regenerate_test_fixtures` work with connect since you need an activated account to use connect.

Reply:
You can create another dummy account and connect it to the "platform" dummy account if you need to. I wasn't really sure where to put this anyways so suggestions are welcome.

Reply:
OK, I'm happy for this to be in `__init__` for now, can be tidied up another time.